microsoft-edge/webview2/release-notes/index.md

```diff
@@ -16,18 +16,309 @@ These Release Notes provide information about new features and bug fixes that ar
 the templates to copy for incoming sections are in ./includes/templates.md
 
 this webpage covers the most recent 6 months; periodically move oldest h2 sections from bottom of present file to archive.md.  eg covers:
+Dec 2025
 Nov 2025
 Oct 2025
 Sep 2025
 Aug 2025
 Jul 2025
-Jun 2025
 
 moved to archive: 
-May 2025
+Jun 2025
 -->
 
 
+<!-- Dec 2025 Release SDK -->
+<!-- ====================================================================== -->
+## 1.0.3650.58
+
+Release Date: December 2, 2025
+
+[NuGet package for WebView2 SDK 1.0.3650.58](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3650.58)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 143.0.3650.58 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions to Phase 3: Stable in Release
+
+The following APIs have been promoted from Phase 2: Stable in Prerelease, to Phase 3: Stable in Release, and are now included in this Release SDK.
+
+
+<!-- ---------- -->
+###### script output including Find APIs & AllowHostInputProcessing
+<!-- todo: delete all these APIs; already reported reaching Phase 3 -->
```

Comment:
delete entire section, b/c already reached Phase 3:
* **Customize the Find behavior (Find API)** section below, August 2025
* AllowHostInputProcessing: **Allow input event messages to pass through the browser window** section below, July 2025

@vootlasaichandu @Ju-MsGit - re-run Release script to get Dec. content?